### PR TITLE
Fix #2314 AbstractDiscovery delayTime calculate error

### LINKED ISSUE #2314: [Bug] AbstractDiscovery delayTime calculate error
### Search before asking

- [X] I searched the [issues](https://github.com/apache/incubator-linkis/issues) and found no similar issues.

### Linkis Component

linkis-commons

### What happened + What you expected to happen

val delayTime = timeUnit.convert(timeUnit.toMillis(period) / 5, timeUnit)

the code timeUnit.convert(timeUnit.toMillis(period) / 5, timeUnit) is equals to timeUnit.toMillis(period) / 5,  

expected the delay is period/5  

### Relevent platform

all

### Reproduction script

no

### Anything else

no

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by jackxu2011**
### What is the purpose of the change
fix #2314 

### Brief change log
(for example:)
- change the delayTime to   if (period < 10) 1 else period / 5
- use ByteTimeUtils.msDurationToString replace Utils.msDurationToString
- format some code

### Verifying this change
(Please pick either of the following options)  
This change is a trivial rework / code cleanup without any test coverage.  

This change added tests and can be verified as follows:  

### Does this pull request potentially affect one of the following parts:
- Dependencies (does it add or upgrade a dependency): (no)
- Anything that affects deployment: (no)
- The MGS(Microservice Governance Services), i.e., Spring Cloud Gateway, OpenFeign, Eureka.: (no)

### Documentation
- Does this pull request introduce a new feature? (no)
- If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

File: linkis-commons/linkis-module/src/main/scala/org/apache/linkis/server/Message.scala [hunk 1]
```diff
@@ -95,13 +94,13 @@ object Message {
...
   implicit def error(t: Throwable): Message = {
-    Message(status =  1).setMessage(ExceptionUtils.getRootCauseMessage(t)) << ("stack", ExceptionUtils.getFullStackTrace(t))
+    Message(status = 1).setMessage(ExceptionUtils.getRootCauseMessage(t)) << ("stack", ExceptionUtils.getStackTrace(t))
   }
   implicit def error(e: (String, Throwable)): Message = error(e._1, e._2)
   implicit def error(msg: String, t: Throwable): Message = {
-    val message = Message(status =  1)
+    val message = Message(status = 1)
     message.setMessage(msg)
-    if(t != null) message << ("stack", ExceptionUtils.getFullStackTrace(t))
```
Comment (peacewong) [inline linkis-commons/linkis-module/src/main/scala/org/apache/linkis/server/Message.scala:104]: What is this replacement for?

File: linkis-commons/linkis-module/src/main/scala/org/apache/linkis/server/conf/ServerConfiguration.scala [hunk 2]
```diff
@@ -65,19 +65,19 @@ object ServerConfiguration extends Logging{
   val BDP_TEST_USER = CommonVars("wds.linkis.test.user", "")
 
   val BDP_SERVER_HOME = CommonVars("wds.linkis.server.home", CommonVars("LINKIS_HOME", "").getValue)
-  val BDP_SERVER_DISTINCT_MODE = CommonVars("wds.linkis.server.distinct.mode", new Boolean(true))
+  val BDP_SERVER_DISTINCT_MODE: CommonVars[lang.Boolean] = CommonVars("wds.linkis.server.distinct.mode", lang.Boolean.TRUE)
```
Comment (peacewong) [inline linkis-commons/linkis-module/src/main/scala/org/apache/linkis/server/conf/ServerConfiguration.scala:68]: Why is there no unified style for Boolean here?

File: linkis-commons/linkis-httpclient/src/main/scala/org/apache/linkis/httpclient/discovery/AbstractDiscovery.scala [hunk 3]
```diff
@@ -159,10 +158,10 @@ abstract class AbstractDiscovery extends Discovery with Closeable with Logging {
       ""
     }
     if (StringUtils.isBlank(updateUnhealthyUrl)) {
-      logger.info(s"${unhealthyUrl}  unhealthy url not exists")
+      info(s"${unhealthyUrl}  unhealthy url not exists")
```
Comment (peacewong) [inline linkis-commons/linkis-httpclient/src/main/scala/org/apache/linkis/httpclient/discovery/AbstractDiscovery.scala:161]: Removing logger is not recommended

File: linkis-commons/linkis-common/src/main/scala/org/apache/linkis/common/utils/VariableUtils.scala [hunk 4]
```diff
@@ -237,7 +236,7 @@ object VariableUtils extends Logging {
               }
             }
             if (!expressionCache.contains(expression)) {
-              logger.info(s"Variable expression [$str] = $res(变量表达式[$str] = $res)")
+              info(s"Variable expression [$str] = $res(变量表达式[$str] = $res)")
```
Comment (peacewong) [inline linkis-commons/linkis-common/src/main/scala/org/apache/linkis/common/utils/VariableUtils.scala:239]: Removing logger is not recommended

File: linkis-commons/linkis-module/src/main/scala/org/apache/linkis/server/conf/ServerConfiguration.scala (hunk 2 above)
Comment (jackxu2011) [reply linkis-commons/linkis-module/src/main/scala/org/apache/linkis/server/conf/ServerConfiguration.scala:68]: this Boolean is used by java,   scala boolean can not be recognition

File: linkis-commons/linkis-hadoop-common/src/main/scala/org/apache/linkis/hadoop/common/utils/HDFSUtils.scala [hunk 5]
```diff
@@ -119,7 +118,9 @@ object HDFSUtils extends Logging {
   def createFileSystem(userName: String, conf: org.apache.hadoop.conf.Configuration): FileSystem =
     getUserGroupInformation(userName)
       .doAs(new PrivilegedExceptionAction[FileSystem] {
-        def run = FileSystem.get(conf)
+        // scalastyle:off FileSystemGet
```
[the remark this answers is not in the document]
Comment (jackxu2011) [reply linkis-commons/linkis-hadoop-common/src/main/scala/org/apache/linkis/hadoop/common/utils/HDFSUtils.scala:121]: this is for ide  style check

File: linkis-commons/linkis-common/src/main/scala/org/apache/linkis/common/utils/VariableUtils.scala (hunk 4 above)
Comment (jackxu2011) [reply linkis-commons/linkis-common/src/main/scala/org/apache/linkis/common/utils/VariableUtils.scala:239]: ok

File: linkis-commons/linkis-module/src/main/scala/org/apache/linkis/server/Message.scala (hunk 1 above)
Comment (peacewong) [reply linkis-commons/linkis-module/src/main/scala/org/apache/linkis/server/Message.scala:104]: ok

**Review by peacewong — APPROVED** [review]
LGTM.

**Pull request merged**